<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(214, 43, 52, 255), stop:1 rgba(251, 175, 178, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>71</y>
      <width>600</width>
      <height>338</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 255, 255);
border-radius: 20px</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>77</y>
       <width>466</width>
       <height>72</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #BA5D5D</string>
     </property>
     <property name="text">
      <string>Please scan your student ID below if you wish to vote for</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblGroupName">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>157</y>
       <width>601</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>70</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #E65353</string>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>REVERSI</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnLogout">
    <property name="geometry">
     <rect>
      <x>723</x>
      <y>27</y>
      <width>44</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 22;
background-color: rgb(154, 154, 154);
color: #CCCCCC</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
